### DISCUSSION TITLE
Challenge 1 Existing Florida Analysis by Shimberg Center

### DISCUSSION THREAD

**Discussion by ryancaralis** [datakind/datakit-housing-fall-2024#48] [Show and tell]
Hi all,

http://www.shimberg.ufl.edu/publications/RMS_2022.pdf

I found the ^ report after looking into @JonathanCote90's dashboard submission and the data source he used. 

The Shimberg Center provides the data they use in the study here: http://flhousingdata.shimberg.ufl.edu/affordability/results?nid=100. 
Most of it is at the county level. I think the report answers the analysis questions and what the Christian Service Center needs.

I'm wondering what would be most helpful going forward. Some things that come to mind:

- download data from Shimberg Center and make custom visuals the Christian Service Center wants
- identify analysis gaps between what the Christian Service Center wants and what's in the Shimberg Center analysis, then fill the gaps with other data sources
- Try to replicate the reports' cross-tabulations and visualizations pulling from the ACS to make the data available for other states / make an up to date version (most recent ACS data) of the report

**Comment by ryancaralis** [comment]
It looks like the report uses custom tabulations made from PUMS data. I'm going to look into sourcing the PUMS data through the Census API this weekend.

**Reply by ryancaralis** [reply]
https://github.com/censusdis/censusdis

^package looks like the best way to access any of the census data. there's a separate tutorial github page and several notebooks that contain examples on how to access different datasets, including PUMS data.

**Comment by PykaAlexandro** [comment]
https://walker-data.com/tidycensus/ is the R equivalent, if anybody is interested

**Comment by ryancaralis** [comment]
I went through the censusdis tutorial and made a reference notebook with examples from the lessons and exercises if anyone is looking for a quick reference. After I reviewed the PUMS data variables, I selected ones related to housing or potential segments, and setup the queries for all of the PUMS areas in Florida.

Next I'm going to subset the query to one county in Florida and try to replicate the custom cross-tabs made by the Shimberg Center, but don't think I'll have time to until the weekend of 11/2. Adding the files here for reference.

[pums_files.zip](https://github.com/user-attachments/files/17453762/pums_files.zip)
